app/tests/v1/test_auth.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+from flask import json
+from app import create_app
+
+class TestAuthentication(unittest.TestCase):
+    """Tests for user authentication"""
+
+    def setUp(self):
+        """Set up for configuration and testing env"""
+        self.app = create_app('testing')
+        self.test_client = self.app.test_client()
+        self.app_context = self.app.app_context()
+        self.client = self.app.test_client()
+
+        self.user_admin = json.dumps({
+            "name": "brian",
+            "email": "brian@gmail.com",
+            "password": "1234",
+            "role": "admin"
+        })
+
+        self.user_attendant = json.dumps({
+                "name": "loise",
+                "email": "loise@gmail.com",
+                "password": "1234",
+                "role": "attendant"
+            })
+        
+        self.login_admin = json.dumps({
+            "email": "brian@gmail.com",
+            "password": "1234"
+        })
+        
+        self.login_attendant = json.dumps({
+            "email": "loise@gmail.com",
+            "password": "1234"
+        })
+        
+
+
+
+    def test_sign_up_user(self):
+       
+        new_user = self.client.post('/api/v1/login', data=self.user_attendant, 
+            content_type='application/json')
+        result = json.loads(new_user.data.decode())
+        self.assertEqual(result['output':'success'])
+        self.assertEqual(new_user.status_code, 201)
+        
+
+    def test_user_login(self):
+        
+        user_attendant_login = self.client.post('/api/v1/login',data=self.login_attendant,
+          content_type = 'application/json') 
+        self.assertEqual(user_attendant_login.status_code, 200)
+        self.token_attendant = json.loads(user_attendant_login.data.decode()).get("x-api-key")
+        
+        
+    
+    def test_user_not_exist(self):
+        """Test that user can login"""
+        
+        user_login = self.client.post('/api/v1/login',data=self.user_attendant,
+          content_type='application/json')
+        response = json.loads(user_login.data.decode())
+        self.assertEqual(response['message'], 'Your account does not exist!, Please Register!')
+        self.assertEqual(user_login.status_code, 401)
```

Comment:
Good job, but also try to test response messages and data instead of only testing for status codes